.gitignore

```
@@ -1,6 +1,10 @@
 *.jl.*.cov
 *.jl.cov
 *.jl.mem
-/Manifest.toml
+Manifest.toml
 /docs/Manifest.toml
 /docs/build/
+
+.vscode
+benchmark
+exports
```

I guess it should be?
```suggestion
benchmarks/**/exports
```